java-parser/src/main/java/org/archcnl/javaparser/visitors/NormalAnnotationExpressionVisitor.java

```diff
@@ -19,8 +19,8 @@
     private AnnotationInstance annotationInstance;
 
     @Override
-    public void visit(NormalAnnotationExpr n, Void arg) {
-        List<AnnotationMemberValuePair> values =
+    public void visit(final NormalAnnotationExpr n, final Void arg) {
```

Comment:
What do the `final` additions have to do with improved test coverage?
They make the history harder to understand and therefore I would suggest splitting these changes off into another PR.

Reply:
1. It isn't test class 2. It is only minor improve within the ticket

Reply:
I don't think that this changes are so big that new PR should be created